/**:
  ros__parameters:
    costmap_frame: "map"
    vehicle_frame: "base_link"
    map_frame: "map"
    update_rate: 10.0
    activate_by_scenario: false
    grid_min_value: 0.0
    grid_max_value: 1.0
    grid_resolution: 0.3
    grid_length_x: 70.0
    grid_length_y: 70.0
    grid_position_x: 0.0
    grid_position_y: 0.0
    maximum_lidar_height_thres: 2.5  # [m] when use_points is true, ignore points with a z value above this (in the vehicle_frame)
    minimum_lidar_height_thres: 0.0  # [m] when use_points is true, ignore points with a z value bellow this (in the vehicle_frame)
    use_wayarea: true
    use_parkinglot: true
    use_objects: true
    use_points: true
    expand_polygon_size: 0.5
    size_of_expansion_kernel: 9
